test/new/db/archos/linux-x64/dbg_bps

```diff
@@ -197,23 +197,27 @@ EOF
 RUN
 
 NAME='valid db'
-FILE=../bins/elf/analysis/x86-helloworld-gcc
+FILE=../bins/elf/analysis/x64-loop
```

Comment:
Why use a different bin?

Reply:
Hello world printed
```
valid
+Hello world!
```

When it was ran